Error: Cannot find module '@angular/cli' when running storybook command in new angular appliction

<!-- Please do your best to fill out all of the sections below! -->

## Current Behavior
<!-- What is the behavior that currently you experience? -->
I got `Error: Cannot find module '@angular/cli'` when I run `yarn nx run nx-ui:storybook`.
see log at Steps to Reproduce.

## Expected Behavior
<!-- What is the behavior that you expect to happen? -->
<!-- Is this a regression? .i.e Did this used to be the behavior at one point?  -->
Should be open the storybook ui.

## Steps to Reproduce
<!-- Help us help you by making it easy for us to reproduce your issue! -->
https://github.com/wizardnet972/nx-storybook
```
$ npx create-nx-workspace
✔ Workspace name (e.g., org name)     · nx-storybook
✔ What to create in the new workspace · empty
✔ Use Nx Cloud? (It's free and doesn't require registration.) · No
$ cd nx-storybook
$ yarn add @nrwl/node @nrwl/angular
$ yarn nx g @nrwl/node:app nx-api
$ yarn nx g @nrwl/angular:app nx-ui
✔ Which stylesheet format would you like to use? · scss
✔ Would you like to configure routing for this application? (y/N) · true
yarn add --dev @nrwl/storybook
yarn nx g @nrwl/angular:storybook-configuration nx-ui
✔ Configure a Cypress e2e app to run against the storybook instance? (Y/n) · false
✔ Automatically generate *.stories.ts files for components declared in this project? (Y/n) · true
✔ Automatically generate *.spec.ts files in the generated Cypress e2e app? (Y/n) · false

$ yarn nx run nx-ui:storybook

yarn run v1.22.11
$ /Users/wizardnet972/code/nx-storybook/node_modules/.bin/nx run nx-ui:storybook

> nx run nx-ui:storybook
info => Loading presets
info => Found custom tsconfig.json
info => Using implicit CSS loaders
Error: Cannot find module '@angular/cli'
Require stack:
- /Users/wizardnet972/code/nx-storybook/node_modules/@storybook/angular/dist/ts3.9/server/framework-preset-angular-cli.js
- /Users/wizardnet972/code/nx-storybook/node_modules/@storybook/core-common/dist/cjs/presets.js
- /Users/wizardnet972/code/nx-storybook/node_modules/@storybook/core-common/dist/cjs/index.js
- /Users/wizardnet972/code/nx-storybook/node_modules/@storybook/core-server/dist/cjs/index.js
- /Users/wizardnet972/code/nx-storybook/node_modules/@storybook/core/dist/cjs/server.js
- /Users/wizardnet972/code/nx-storybook/node_modules/@storybook/core/server.js
- /Users/wizardnet972/code/nx-storybook/node_modules/@nrwl/storybook/src/executors/storybook/storybook.impl.js
- /Users/wizardnet972/code/nx-storybook/node_modules/@nrwl/tao/src/shared/workspace.js
- /Users/wizardnet972/code/nx-storybook/node_modules/@nrwl/tao/src/commands/run.js
- /Users/wizardnet972/code/nx-storybook/node_modules/@nrwl/tao/index.js
- /Users/wizardnet972/code/nx-storybook/node_modules/@nrwl/cli/lib/run-cli.js


          Broken build, fix the error above.
          You may need to refresh the browser.


———————————————————————————————————————————————

>  NX   ERROR  Running target "nx-ui:storybook" failed

  Failed tasks:

  - nx-ui:storybook

  Hint: run the command with --verbose for more details.

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

<!-- Can you reproduce this on https://github.com/nrwl/nx-examples? -->
<!-- If so, open a PR with your changes and link it below. -->
<!-- If not, please provide a minimal Github repo -->
<!-- At the very least, provide as much detail as possible to help us reproduce the issue -->

### Failure Logs
<!-- Please include any relevant log snippets or files here. -->

### Environment
<!-- It's important for us to know the context in which you experience this behavior! -->
<!-- Please paste the result of `nx report` below! -->
```
  NX  Report complete - copy this into the issue template

  Node : 16.13.0
  OS   : darwin arm64
  yarn : 1.22.11

  nx : 13.2.2
  @nrwl/angular : 13.2.2
  @nrwl/cli : 13.2.2
  @nrwl/cypress : 13.2.2
  @nrwl/devkit : 13.2.2
  @nrwl/eslint-plugin-nx : 13.2.2
  @nrwl/express : undefined
  @nrwl/jest : 13.2.2
  @nrwl/linter : 13.2.2
  @nrwl/nest : undefined
  @nrwl/next : undefined
  @nrwl/node : 13.2.2
  @nrwl/nx-cloud : undefined
  @nrwl/react : undefined
  @nrwl/react-native : undefined
  @nrwl/schematics : undefined
  @nrwl/tao : 13.2.2
  @nrwl/web : undefined
  @nrwl/workspace : 13.2.2
  @nrwl/storybook : 13.2.2
  @nrwl/gatsby : undefined
  typescript : 4.4.4
  rxjs : 7.4.0
  ---------------------------------------
  Community plugins:
  	 @angular/animations: 13.0.2
  	 @angular/common: 13.0.2
  	 @angular/compiler: 13.0.2
  	 @angular/core: 13.0.2
  	 @angular/forms: 13.0.2
  	 @angular/platform-browser: 13.0.2
  	 @angular/platform-browser-dynamic: 13.0.2
  	 @angular/router: 13.0.2
  	 @angular-devkit/build-angular: 13.0.3
  	 @angular/compiler-cli: 13.0.2
  	 @angular/language-service: 13.0.2
  	 @storybook/angular: 6.4.0-rc.7

(node:33567) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./" in the "exports" field module resolution of the package at /Users/wizardnet972/code/nx-storybook/node_modules/tslib/package.json.
Update this package.json to use a subpath pattern like "./*".
(Use `node --trace-deprecation ...` to show where the warning was created)
✨  Done in 0.85s.
```

--- a/packages/angular/src/generators/init/init.spec.ts
+++ b/packages/angular/src/generators/init/init.spec.ts
@@ -33,6 +33,7 @@ describe('init', () => {
     expect(dependencies['@angular/router']).toBeDefined();
     expect(dependencies['rxjs']).toBeDefined();
     expect(dependencies['zone.js']).toBeDefined();
+    expect(devDependencies['@angular/cli']).toBeDefined();
     expect(devDependencies['@angular/compiler-cli']).toBeDefined();
     expect(devDependencies['@angular/language-service']).toBeDefined();
     expect(devDependencies['@angular-devkit/build-angular']).toBeDefined();

--- a/packages/angular/src/generators/init/init.ts
+++ b/packages/angular/src/generators/init/init.ts
@@ -95,6 +95,7 @@ function updateDependencies(host: Tree): GeneratorCallback {
       'zone.js': '~0.11.4',
     },
     {
+      '@angular/cli': angularDevkitVersion,
       '@angular/compiler-cli': angularVersion,
       '@angular/language-service': angularVersion,
       '@angular-devkit/build-angular': angularDevkitVersion,

